### PR TITLE
Update Alpine Lima ISO to 0.2.21

### LINKED ISSUE #2771: Add git to VM
### Problem Description

I would like to be able to set the build context for an image in a Docker Compose file to a git repo URL.  This does not currently work because git is not installed in the VM.

### Proposed Solution

Install git in the VM. @ryanfb has helpfully linked this issue to alpine-lima.

### Additional Information

Platform: arm64

### PR DESCRIPTION
**Pull request by ryanfb**
Because this release incorporates lima-vm/alpine-lima#84, this should fix #2771

### CODE REVIEW

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Comment by jandubois** [comment]
Note that #2864 should be addressed before this PR is merged

**Review by jandubois — CHANGES_REQUESTED** [review]
Please fix linter complaints.

Also wait for a new WSL-distro release to keep the Windows version in sync with the macOS/Linux one. I've just created https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/pull/49. Once it is merged, I will create a new release there as well.

**Comment by jandubois** [comment]
> Please fix linter complaints.

I'm sorry, the linter errors are not due to your PR; will have to be fixed in main, and then you have to rebase. Not sure how this could happen.

**Comment by ryanfb** [comment]
> > Please fix linter complaints.
> 
> I'm sorry, the linter errors are not due to your PR; will have to be fixed in main, and then you have to rebase. Not sure how this could happen.

Happy to rebase and push once #2895 is merged.

**Comment by jandubois** [comment]
> Happy to rebase and push once #2895 is merged.

This is not necessary; sorry I was severely confused earlier.

However, we do want the WSL-distro changes included in this PR, to keep it in sync with alpine-lima. I think I've sorted this out now, and will ping you here once we have a new release for WSL-distro.

**Comment by jandubois** [comment]
> will ping you here once we have a new release for WSL-distro.

I've made a WSL-distro release with the corresponding changes and pushed an extra commit to your PR to bump that image as well. Will merge if CI ends up being green (which it should).

**Review by jandubois — APPROVED** [review]
Thanks!

**Pull request merged**
